package.json

```diff
@@ -40,31 +45,46 @@
     "typeface-roboto": "^1.0.0"
   },
   "devDependencies": {
-    "@neutrinojs/airbnb": "9.0.0",
-    "@neutrinojs/jest": "9.0.0",
-    "@neutrinojs/react": "9.0.0",
+    "@babel/core": "^7.26.0",
+    "@babel/eslint-parser": "^7.25.9",
+    "@babel/plugin-transform-runtime": "^7.25.9",
+    "@babel/preset-env": "^7.26.0",
+    "@babel/preset-react": "^7.26.3",
+    "@babel/runtime": "^7.26.0",
     "@pollyjs/adapter-fetch": "^5.0.0",
     "@pollyjs/adapter-node-http": "^5.0.0",
     "@pollyjs/core": "^5.0.0",
     "@pollyjs/persister-fs": "^5.0.0",
-    "@testing-library/react": "10.0.6",
+    "@testing-library/react": "^16.1.0",
+    "babel-loader": "^9.2.1",
+    "babel-preset-jest": "^29.6.3",
     "codecov": "3.7.2",
-    "eslint": "7.20.0",
+    "eslint": "^8.0.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jest": "27.0.0",
+    "eslint-plugin-jest-dom": "^5.5.0",
+    "eslint-plugin-react": "^7.37.3",
+    "eslint-plugin-testing-library": "^7.1.1",
```

Comment:
note that you're not loading the eslint plugin for testing-libary atm (but that's OK, you can always enable it later)